tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -413,16 +413,16 @@ def print_perf_counters(perf_counts_list):
         total_time_cpu = timedelta()
         logger.info(f"Performance counts for {ni}-th infer request")
         for pi in perf_counts:
-            print(f"{pi.node_name[:max_layer_name - 4] + '...' if (len(pi.node_name) >= max_layer_name) else pi.node_name:<30}"
-                                                                f"{str(pi.status):<15}"
-                                                                f"{'layerType: ' + pi.node_type:<30}"
-                                                                f"{'realTime: ' + str((pi.real_time // timedelta(microseconds=1)) / 1000.0):<20}"
-                                                                f"{'cpu: ' +  str((pi.cpu_time // timedelta(microseconds=1)) / 1000.0):<20}"
-                                                                f"{'execType: ' + pi.exec_type:<20}")
+            print(f"{pi.node_name[:max_layer_name - 4] + '...' if (len(pi.node_name) >= max_layer_name) else pi.node_name:<30} "
+                                                                f"{str(pi.status):<15} "
+                                                                f"layerType: {str(pi.node_type):<15} "
+                                                                f"execType: {pi.exec_type:<15} "
+                                                                f"realTime (ms): {float(int(pi.real_time // timedelta(microseconds=1)) / 1000.0):.6f} "
+                                                                f"cpuTime (ms): {float(int(pi.cpu_time // timedelta(microseconds=1)) / 1000.0):.6f}")
             total_time += pi.real_time
             total_time_cpu += pi.cpu_time
-        print(f'Total time:     {(total_time // timedelta(microseconds=1)) / 1000.0} seconds')
-        print(f'Total CPU time: {(total_time_cpu // timedelta(microseconds=1)) / 1000.0} seconds\n')
+        print(f'Total time:     {int(total_time // timedelta(microseconds=1)) / 1000.0} seconds')
+        print(f'Total CPU time: {int(total_time_cpu // timedelta(microseconds=1)) / 1000.0} seconds\n')
```

Comment:
I don't know why this is printing "seconds" as the unit. A `timedelta() // timedelta(microseconds=1)` object is an `int` representing microseconds. Divide that by `1000.0`, and you have milliseconds. That was the old code, until it was changed in https://github.com/openvinotoolkit/openvino/pull/13743, for some unknown reason - perhaps @Wovchena can explain? 

Anyway, if you want milliseconds (to make it the same as everywhere else) then use `total_time / timedelta(milliseconds=1)`. If you want seconds, use `total_time / timedelta(seconds=1)`.

Reply:
> That was the old code, until it was changed in https://github.com/openvinotoolkit/openvino/pull/13743, for some unknown reason - perhaps @Wovchena can explain?

The print format was HH:MM:SS.XXXXXX. It felt more reasonable to say that this is seconds, because it's the last value before decimal point.

Reply:
> The print format was HH:MM:SS.XXXXXX. It felt more reasonable to say that this is seconds, because it's the last value before decimal point.

The print format of `(total_time // timedelta(microseconds=1)) / 1000.0` is a `float`, and it represents milliseconds, since `total_time // timedelta(microseconds=1)` is an `int` representing microseconds, and this value divided by `1000.0` is a `float` result representing milliseconds.

Reply:
Indeed it's milliseconds, renamed!